src/advanced/node-management.md

```diff
@@ -543,12 +543,12 @@ Returns the headers of the consecutive `count` blocks up to the `latest` height.
 - **count**: integer  
   The number of blocks to return. Should not be greater than
   [`MAX_BLOCKS_PER_REQUEST`][github_explorer]
-- **skip_empty_blocks**: bool  
-  If `true`, then only non-empty blocks are returned
-- **latest**: integer  
+- **skip_empty_blocks**: bool=  
+  If `true`, then only non-empty blocks are returned. Default value is `false`
```

Comment:
*The* default value?

Reply:
Uhm, I am not a native speaker, always have troubles with (the? : ) articles